config/routes.rb

```diff
@@ -2,5 +2,5 @@
   # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
   root 'games#index'
   resources :users, :sessions
-  get 'games/:id' => 'games#show'
+  get 'games/:id' => 'games#show', as: :games
```

Comment:
Ed - I am going to merge, but just curious -- did you add the as: games just for consistency so the path was given a prefix - in case someone edited the code and would be expecting that prefix?